src/output-writer/google-sheets-writer.ts

```diff
@@ -1,3 +1,614 @@
+        },
+      },
+    },
+  ]);
+
+  const sheetId = updateResponse.data.replies[0].addSheet.properties.sheetId;
+  const auditBreakdownSheetId =
+    updateResponse.data.replies[1].addSheet.properties.sheetId;
+  const runInfoSheetId =
+    updateResponse.data.replies[2].addSheet.properties.sheetId;
+
+  await batchUpdate(service, spreadsheetId, [
+    {
+      deleteSheet: {
+        // This is the initial sheet, "Sheet 1" that is created when the google sheet is created
+        sheetId: spreadsheet.data.sheets[0].properties.sheetId,
+      },
+    },
+    {
+      // URL column width in main sheet
+      updateDimensionProperties: {
+        range: {
+          sheetId,
+          dimension: 'COLUMNS',
+          startIndex: 0,
+          endIndex: 1,
+        },
+        properties: {
+          pixelSize: 400,
+        },
+        fields: 'pixelSize',
+      },
+    },
+    {
+      // First column in run info sheet
+      updateDimensionProperties: {
+        range: {
+          sheetId: runInfoSheetId,
+          dimension: 'COLUMNS',
+          startIndex: 0,
+          endIndex: 1,
+        },
+        properties: {
+          pixelSize: 200,
+        },
+        fields: 'pixelSize',
+      },
+    },
+    {
+      // Center-align header rows
+      repeatCell: {
+        range: {
+          sheetId,
+          startRowIndex: 0,
+          endRowIndex: 2,
+          startColumnIndex: 0,
+        },
+        cell: {
+          userEnteredFormat: {
+            horizontalAlignment: 'CENTER',
+          },
+        },
+        fields: 'userEnteredFormat.horizontalAlignment',
+      },
+    },
+    {
+      // Don't wrap first column (URLs)
+      repeatCell: {
+        range: {
+          sheetId,
+          startRowIndex: 2,
+          startColumnIndex: 0,
+          endColumnIndex: 1,
+        },
+        cell: {
+          userEnteredFormat: {
+            wrapStrategy: 'CLIP',
+          },
+        },
+        fields: 'userEnteredFormat.wrapStrategy',
+      },
+    },
+  ]);
+
+  return {
+    async writeRunInfo(runInfo) {
+      const valuesBatchUpdateRequest: Sheets.sheets_v4.Schema$BatchUpdateValuesRequest =
+        {
+          data: [
+            {
+              range: `'${sheetNames.runInfo}'!A1`,
+              values: [
+                ['Command', runInfo.command],
+                ['Time', runInfo.time],
+                [],
+                ['node version:', runInfo.versions.node],
+                ['npm version:', runInfo.versions.npm],
+                ['lighthouse version:', runInfo.versions.lighthouse],
+                [
+                  'lighthouse-parade version:',
+                  runInfo.versions.lighthouseParade,
+                ],
+                ['chrome version:', runInfo.versions.chrome],
+                [],
+                ['OS:', runInfo.system.operatingSystem],
+                ['Memory:', runInfo.system.memory],
+                ['CPU Info', runInfo.system.cpus],
```

Comment:
Ahh I may be confused. It does give me this info: "(8) Apple M1"

In the past I've had to manually switch from run (emulated?) x64 locally to arm64 so it might be nice to include which version is running? I'm not sure how many people that would be relevant for though

Reply:
Since it is a property of the node binary being installed and used, not of the computer's processors

Reply:
Makes sense :+1: